packages/fast-components/src/name-tag/fixtures/mark.html

```diff
@@ -1,4 +1,6 @@
 <fast-design-system-provider>
-    <fast-name-tag>Mark!</fast-name-tag>
-    <fast-name-tag style="--background-color: #00F">Blue Mark!</fast-name-tag>
+    <fast-design-system-provider>

```

Comment:
does this need to be nested twice?

Reply:
I was testing inheritance. It's not a "real" component but I can revert